discovery/src/main/java/com/ibm/watson/developer_cloud/discovery/v1/model/QueryNoticesOptions.java

```diff
@@ -147,6 +122,36 @@ public Builder addReturnField(String returnField) {
     }
 
     /**
+     * Adds an sort to sort.
+     *
+     * @param sort the new sort
+     * @return the QueryNoticesOptions builder
+     */
+    public Builder addSort(String sort) {
+      Validator.notNull(sort, "sort cannot be null");
+      if (this.sort == null) {
+        this.sort = new ArrayList<String>();
+      }
+      this.sort.add(sort);
+      return this;
+    }
+
+    /**
+     * Adds an passagesFields to passagesFields.
+     *
```

Comment:
This is another place where we might want to use `x-item-name`, probably just "passagesField".